_data/sites.json

```
@@ -9755,6 +9755,16 @@
         ]
     },
 
+    {
+        "name": "JEDEC",
+        "url": "https://www.jedec.org/user",
+        "difficulty": "medium",
+        "notes": "Edit -> Cancel account, then click the link sent to your email.",
```

```suggestion
        "difficulty": "easy",
        "notes": "Edit ⮕ Cancel account, then click the link sent to your email.",
```
I think this still sounds easy, since it's just a couple simple steps, so I'll change it accordingly